examples/webhooks/index.js

```diff
@@ -25,7 +25,7 @@ app.get('/', function (req, res) {
 
 app.post('/webhook', verifyMiddleware, function (req, res) {
   // The message is authentic:
-  console.log("The message is authentic:\n" + JSON.stringify(req.body, null, '  '));
+  console.log("The message is authentic:\n" + JSON.stringify(req.data, null, '  '));
```

Comment:
Woops, you're right. Fixed https://github.com/SudoPlz/acuity-js/commit/7fa2696343b0e464f888cb4afe045d838c48db20